src/main/java/apprenticeship/Board.java

```diff
@@ -3,34 +3,38 @@
 import java.util.Arrays;
 
 public class Board {
-    private char cells[];
-    private int numberOfCells = 9;
+    private String[] cells = {"1", "2", "3", "4", "5", "6", "7", "8", "9"};
```

Comment:
You don't need to make any changes for this comment - just something to think about. 

Right now, the `cells` array is very closely coupled to the `getGrid` method. The `getGrid` method literally reads values from the array and prints them to the screen (with some added dashes and slashes). However, suppose I wanted you to change the way you print out the board - maybe I want you to start the board from index zero instead of one. Maybe I want to be able to play a three or four player game. Maybe I want the board to be 4x4 instead of 3x3. You would have to change both the `getGrid` method (i.e. the way you present the data) and the cells array (the data structure itself). This is generally not a good thing - you want the data structure to be independent of the way you display it to the screen.

Maybe think about some ways that you could decouple the data and the display? Do you need to store cell strings? Is there maybe another way you could store this data?